clang/lib/Parse/ParsePragma.cpp

```diff
@@ -568,6 +574,11 @@ void Parser::initializePragmaHandlers() {
   MaxTokensTotalPragmaHandler = std::make_unique<PragmaMaxTokensTotalHandler>();
   PP.AddPragmaHandler("clang", MaxTokensTotalPragmaHandler.get());
 
+  if (getLangOpts().ZOSExt) {
```

Comment:
This isn't particularly ZOS specific other than for compiler compat, right?  Is there a good reason we couldn't allow this syntax on other compilers?

Reply:
IDK why I didn't see the comment response here... I think doing these based on the target here is somewhat reasonable, it harms portability of this attribute, but in a way that I don't think is particularly challenging thanks to the ZOSExt flag being required.